packages/fxa-auth-server/docs/swagger/shared/descriptions.ts

```diff
@@ -180,9 +180,17 @@ const DESCRIPTIONS = {
   productMetadata:
     'Set of key-value pairs used to store additional information about the product. For more information, see [Ecosystem Platform](https://mozilla.github.io/ecosystem-platform/tutorials/subscription-platform#stripe-product-metadata)',
   productName: 'The name of the product purchased.',
+  promotionAmountOff:
+    'Amount (in the currency specified) that will be taken off the subtotal of any invoices for this customer.',
```

Comment:
Not sure if I'm reading this correctly. Is this an amount saying I'd save $2.99 Canadian. Or is this a number of cents like the `amount` field above:
```
(e.g. 100 cents to charge $1.00 or 100 to charge ¥100, a zero-decimal currency)
```

Reply:
As discussed, `amount_off` is in cents.